customize.dist/translations/messages.pl.js

```diff
@@ -0,0 +1,229 @@
+    out.policy_howweuse = 'Jak wykorzystujemy zebraną wiedzę';
+    out.policy_howweuse_p1 = 'Dzieki tym informacjom możemy podejmować lepsze decyzje przy promocji CryptPad, poprzez ocenę które z podjętych przez nas prób okazały się udane. Informacja o twojej lokalizacji daje nam znać, czy powinniśmy zapewnić lepsze wsparcie dla języków poza Angielskim.';
+    out.policy_howweuse_p2 = "Informacje o twojej przeglądarce (czy jest to aplikacja desktopowa, czy działająca na systemie mobilnym) pozwalają nam na decydowanie przy priorytezowaniu ulepszeń funkcji. Nasz zespół deweloperski jest mały, a my staramy się dokonywać wyborów które poprawią doświadczenia jak największej liczby użytkowników.";
+    out.policy_whatwetell = 'Jakie dane przekazujemy innym';
+    out.policy_whatwetell_p1 = 'Nie dostarczamy osobom trzecim żadnych danych które udało się nam zebrać, lub tych które nam przekazałeś sam, dopóki nie jesteśmy do tego zobligowani prawnie.';
+    out.policy_links = 'Adresy innych stron';
+    out.policy_links_p1 = 'Ta witryna zawiera łącza do innych stron, włączając w to te stworzone przez inne organizacje. Nie jesteśmy odpowiedzialni za praktyki dotyczące prywatności oraz zawartość usługodawców poza tą witryną. Jako główną zasadę przyjmujemy, że łącza do stron zewnętrznych uruchamiane są w nowej karcie lub oknie, aby upewnić cię iż opuszczasz Cryptpad.';
+    out.policy_ads = 'Promocja i reklama';
+    out.policy_ads_p1 = 'Nie wyświetlamy żadnej zawartości promocyjnej online, choć możemy udostępniać łącza do podmiotów finansujących nasze badania.';
+    out.policy_choices = 'Co możesz zrobić';
+    out.policy_choices_open = 'Nasz kod jest open source, więc zawsze masz możliwość hostowania swojej własnej wersji Cryptpad.';
+    out.policy_choices_vpn = 'Jeżeli chcesz korzystać z wersji udostępnianej przez nas, lecz nie chcesz pokazywać swojego adresu IP, możesz chronić swój adres wykorzystując <a href="https://www.torproject.org/projects/torbrowser.html.en" title="downloads from the Tor project" target="_blank" rel="noopener noreferrer">przeglądarki Tor</a>, lub <a href="https://riseup.net/en/vpn" title="VPNs provided by Riseup" target="_blank" rel="noopener noreferrer">VPN</a>.';
+    out.policy_choices_ads = 'Masz również możliwość blokady naszej platformy analitycznej wykorzystując narzędzia adblock, takie jak <a href="https://www.eff.org/privacybadger" title="download privacy badger" target="_blank" rel="noopener noreferrer">Privacy Badger</a>.';
+
+    // terms.html
+
+    out.tos_title = "Warunki korzystania z usług Cryptpad";
+    out.tos_legal = "Prosimy nie bądź złośliwy, obelżywy i nie wykorzystuj tego oprogramowania do cełow niezgodnych z prawem.";
+    out.tos_availability = "Mamy nadzieję iż uznasz tą usługę za przydatną, lecz dostępność i wydajność nie mogą być przez nas gwarantwane. Prosimy, eksportuj swoje dane regularnie.";
```

Comment:
Regarding exporting, I believe that "Prosimy, abyś eksportował swoje dane regularnie." preserves the context well, while sounding more natural.